.github/workflows/inspektor-gadget.yml

```diff
@@ -530,7 +531,7 @@ jobs:
     - name: Build example binaries
       run: |
         for dir in $(dirname $(find examples/gadgets -name README.md)); do
-          go build ./$dir;
+          go build -tags "${{ env.BUILDTAGS }}" ./$dir;
```

Comment:
Ah thanks for catching it. I will update the PR. I was thinking if there is a way to avoid introducing all these build tags and it seems [podman api](https://docs.podman.io/en/latest/_static/api.html) is simple:

```
$ curl --unix-socket /run/podman/podman.sock http://d/v4.0.0/libpod/containers/json
[{"AutoRemove":false,"Command":null,"Created":"2023-05-08T12:18:06.652224626+02:00","CreatedAt":"","Exited":false,"ExitedAt":-62135596800,"ExitCode":0,"Id":"79fc5967c6585b409443dc3a3bb82fcf6b25ef1ac33513eaaeae080fbbed6b14","Image":"localhost/podman-pause:4.5.0-0","ImageID":"24ba8382d076c5a49417dddef16eee88ac38dd5b9480f80a1e8fe6a1120c62d2","IsInfra":true,"Labels":{"io.buildah.version":"1.30.0"},"Mounts":[],"Names":["f77561f23a35-infra"],"Namespaces":{},"Networks":["podman"],"Pid":108601,"Pod":"f77561f23a354b93a2f43ee63f11b5f2d903274297b919da09edbf47af660632","PodName":"test","Ports":null,"Size":null,"StartedAt":1683541144,"State":"running","Status":""}]
...
```
Would need our own http client for list/inspect containers. What do you think about it?

Reply:
> Would need our own http client for list/inspect containers. What do you think about it?

I don't know what's best between the 2 approaches. I can accept either.

If we implement our own http client for this, we could implement a timeout in order to avoid accidental deadlock.

Reply:
Makes sense. Using the API directly simplifies the changes and it isn't a lot of work. Also, I was able to test the timeout to avoid accidental deadlock.